cpp/memilio/math/adapt_rk_fast.h

```diff
@@ -0,0 +1,60 @@
+/* 
+* Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+*
+* Authors: Martin J. Kuehn, Daniel Abele, Rene Schmieding
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#ifndef ADAPT_RK_FAST_H_
+#define ADAPT_RK_FAST_H_
+
+#include "memilio/math/adapt_rk.h"
+
+namespace mio
+{
+
+class FastRKIntegratorCore : public RKIntegratorCore
```

Comment:
If I see this right, the diference between RKIntegratorCore and FastRKIntegratorCore is the convergence criterion, the former checks every compartment individually, the latter checks the maximum. This difference (or any other difference that I missed) must be documented somewhere.

At least add a doc string to the FastRKIntegratorCore class that explains the difference. Also try to find a name for the class that reflects the difference.